rosetta/text/vw_helpers.py

```diff
@@ -147,8 +155,8 @@ def find_start_line_lda_predictions(predictions_file, num_topics):
     return start_line
 
 
-def parse_lda_predictions(
-    predictions_file, num_topics, start_line, normalize=True):
+def parse_lda_predictions(predictions_file, num_topics, start_line,
```

Comment:
Oh wait...now I remember that default values are not appropriate for this function.  Ignore previous comment